spec/system/add_search_rspec.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
```

Comment:
The file name's suffix should be `_spec`. (not `_rspec`)
I think it's just a typo or something though.

Reply:
@jp-ryuji :sparkles: